packages/sdk/src/oracle/services/statemachines/pollNewEvents.ts

```diff
@@ -30,21 +30,30 @@ export function Handlers(store: Store): GenericHandlers<Params, Memory> {
       const currentBlock = memory.lastBlock || startBlock || latestBlock;
       memory.error = undefined;
       try {
-        // dont worry about querying if latest and current are the same
+        // dont worry about querying if latest and current are the same. This optimization causes wierd effects
+        // during testing, as block number rarely changes unless user moves it manually, giving the impression that events are missed.
         if (latestBlock !== currentBlock) {
           // this pulls all events from current to latest block
           await update.oracleEvents(chainId, currentBlock, latestBlock);
           // update our request table list with all known events
           await update.sortedRequests(chainId);
+
+          try {
+            // we can just try to update the current active request, we dont care if it fails, active request might not be set
+            await update.activeRequestFromEvents();
+          } catch (err) {
+            // do nothing
```

Comment:
Can we use the same strategy to drop this ignored error?

Reply:
im going to make a card to refactor specifically for this change, as i need to go through and change it in many places: https://app.shortcut.com/uma-project/story/4475/replace-ignored-errors-with-ignore-existence-errors-in-oo-client-sdk